### DISCUSSION TITLE
How to configure export so that it only exports .cls files?

### DISCUSSION THREAD

**Discussion by jkorhone** [intersystems-community/vscode-objectscript#844] [Q&A] [answered]
I am wondering how to configure export so that it only exports .cls files (e.g. no .ini files)?

**Comment by daimor** [comment, marked as answer]
The export from Explorer menu supposed to work this way, just select classes folder, or particular package and do export with context menu

**Reply by jkorhone** [reply]
Weird now it seems to work that way, not sure what was wrong earlier today :)
